utils/general.py

```diff
@@ -436,6 +436,9 @@ def check_file(file, suffix=''):
             torch.hub.download_url_to_file(url, file)
             assert Path(file).exists() and Path(file).stat().st_size > 0, f'File download failed: {url}'  # check
         return file
+    elif file.startswith('clearml://'):
+        # ClearML Dataset ID
+        return file
     else:  # search
```

Comment:
We should probably prepare for the edge case where user passes clearml file but clearml is not installed.

Reply:
You're 100% correct, thanks for catching it!